<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSeedingEditor</class>
 <widget class="QDialog" name="DlgSeedingEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Define seeding</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="SeedingListWidget" name="lwSeeding">
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Move players</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QPushButton" name="btnUp">
                <property name="text">
                 <string>▲</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDown">
                <property name="text">
                 <string>▼</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbShuffle">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Shuffle players</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
                <item>
                 <widget class="QRadioButton" name="rbShuffleAll">
                  <property name="text">
                   <string>All positions</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">bgSuffleRange</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
                  <property name="sizeConstraint">
                   <enum>QLayout::SetDefaultConstraint</enum>
                  </property>
                  <item>
                   <widget class="QRadioButton" name="rbShuffleRange">
                    <property name="text">
                     <string>From</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">bgSuffleRange</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="sbRangeMin">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="btnShuffle">
                  <property name="text">
                   <string>Shuffle Now</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="SimpleReportLib::SimpleReportViewer" name="repView" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOkay">
       <property name="text">
        <string>Okay</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimpleReportLib::SimpleReportViewer</class>
   <extends>QWidget</extends>
   <header location="global">SimpleReportGeneratorLib/SimpleReportViewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SeedingListWidget</class>
   <extends>QTableWidget</extends>
   <header>ui/SeedingListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgSeedingEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOkay</sender>
   <signal>clicked()</signal>
   <receiver>DlgSeedingEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDown</sender>
   <signal>clicked()</signal>
   <receiver>DlgSeedingEditor</receiver>
   <slot>onBtnDownClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnShuffle</sender>
   <signal>clicked()</signal>
   <receiver>DlgSeedingEditor</receiver>
   <slot>onBtnShuffleClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUp</sender>
   <signal>clicked()</signal>
   <receiver>DlgSeedingEditor</receiver>
   <slot>onBtnUpClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bgSuffleRange</sender>
   <signal>buttonClicked(int)</signal>
   <receiver>DlgSeedingEditor</receiver>
   <slot>onShuffleModeChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lwSeeding</sender>
   <signal>currentCellChanged(int,int,int,int)</signal>
   <receiver>DlgSeedingEditor</receiver>
   <slot>onSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnUpClicked()</slot>
  <slot>onBtnDownClicked()</slot>
  <slot>onBtnShuffleClicked()</slot>
  <slot>onShuffleModeChange()</slot>
  <slot>onSelectionChanged()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="bgSuffleRange"/>
 </buttongroups>
</ui>
